ui/client/components/graph/node-modal/BaseModalContent.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import PropTypes from "prop-types"
+import ModalRenderUtils from "./ModalRenderUtils"
+
+export default function BaseModalContent(props) {
+
+  const {edge, edgeErrors, readOnly, isMarked, changeEdgeTypeValue, children} = props
+
+  return (
+    <div className="node-table">
+      {ModalRenderUtils.renderOtherErrors(edgeErrors, "Edge has errors")}
```

Comment:
This is alsow ugly way.. We shoudl have there another separeted component.. I mean all method from ModalRenderUtils should be in package modal/notifications/ Info / Warning / Error - alsow it looks like we can remove methods: renderWarning and renderInfo.

Reply:
I agree that these piece of code need some refactoring. Think it should be done in separated PR.
What errors have in common with expression :P?